debian/mumbledj.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+After=mumble-server.service
+
+[Service]
+Type=simple
+Restart=always
+ExecStart=/bin/false
+User=mumbledj
+Environment=XDG_CACHE_HOME=/var/cache/mumbledj
```

Comment:
While I think about it, it would be nice if mumbledj itself would place its cache directory within `XDG_CACHE_HOME`; that way I can drop the `~mumbledj/.mumbledj/songs` symlink. See [basedir-spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.8.html) for more information.